src/wp-admin/widgets-form.php

```diff
@@ -211,7 +211,7 @@
 			$id_base      = implode( '-', $pieces );
 			$widget       = get_option( 'widget_' . $id_base );
 			unset( $widget[ $multi_number ] );
-			update_option( 'widget_' . $id_base, $widget );
+			update_option( 'widget_' . $id_base, $widget, false );
```

Comment:
See above, let's leave this for https://core.trac.wordpress.org/ticket/42743.

Reply:
yes